src/version.php

```diff
@@ -3,5 +3,6 @@
     define('WOVN_PHP_NAME', 'WOVN.php');
 }
 if (!defined('WOVN_PHP_VERSION')) {
-    define('WOVN_PHP_VERSION', '0.1.23');
+    $version = isset($_ENV['WOVN_ENV']) && $_ENV['WOVN_ENV'] === 'development' ? 'VERSION' : '0.1.25';
```

Comment:
Let's upgrade to `1.0.0`.
This library is not beta version now.

Reply:
i'm thinking just do this version as-is. Then a separate PR to make version 1.0.0